data/getting-started/pachyderm.md

```diff
@@ -21,19 +21,118 @@ Read about how our Customers leveraged Pachyderm to face their ML/AI challenges
 
-## Installing Pachyderm Operator
-### Prerequisites
-Before installing Pachyderm's Operator, read about Pachyderm's [main concepts](https://docs.pachyderm.com/latest/concepts/) in our documentation.
-Additionally, you will interact with Pachyderm using `pachctl` (Pachyderm's command-line tool) from the cells of your notebooks. Read through our written demo [Open CV](https://docs.pachyderm.com/latest/getting_started/beginner_tutorial/) to familiarize yourself with the main commands. 
+## Before Installing Pachyderm Operator
+
+Before you start the operator installation process, you will need to:
+
+- Familiarize yourself with Pachyderm's **[main concepts](https://docs.pachyderm.com/latest/concepts/)**, and Pachyderm's command-line tool `pachctl`. You will interact with Pachyderm using `pachctl` from the cells of your notebooks. Read through our written demo [Open CV](https://docs.pachyderm.com/latest/getting_started/beginner_tutorial/) to get used to the main commands. 
+- Prepare the installation of Pachyderm by creating an **external object store** (Pachyderm supports all s3-compatible storage solutions) for your data. Make sure to match your cluster's region.
+
+    * Ceph Nano users: Retrieve the `Cluster IP` of your service (in Networking > Services),  your `AWS_ACCESS_KEY_ID` (base 64), and `AWS_SECRET_ACCESS_KEY` (base 64).
+    
+    * AWS users: Retrieve the `arn` of your S3 bucket, create a User, then add an inline policy to grant this user a set of permissions on this bucket.
+
+        Copy/Paste the following in the JSON tab of the policy. Replace <arn:AWS:...> with the arn of your bucket.
+        ``` yaml
+            {
+                "Version": "2012-10-17",
+                "Statement": [
+                    {
+                        "Effect": "Allow",
+                        "Action": [
+                            "s3:ListBucket"
+                        ],
+                        "Resource": [
+                            "<arn:aws:...>"
+                        ]
+                    },
+                    {
+                        "Effect": "Allow",
+                        "Action": [
+                            "s3:PutObject",
+                            "s3:GetObject",
+                            "s3:DeleteObject"
+                        ],
+                        "Resource": [
+                            "<arn:aws:...>/*"
+                        ]
+                    }
+                ]
+            }
+        ```
+
+
+
+- In Red Hat OpenShift Console, we will guide you through the creation of a new namespace for your cluster as well as a secret holding your Bucket's credentials.
+
+    * **A- Create a Project**
+    On the left menu, select Operator > OperatorHub.
+    On the top of the page, click on the Project Dropdown.
+    Choose a project name, then click Create.
 
-### Install the Operator
+    * **B- Create the Namespace in which you will deploy Pachyderm**        
```

Comment:
I think users just need to create a Project in OCP console and a namespace should be automatically created